src/resin-mixpanel-client.js

```diff
@@ -110,6 +110,22 @@ module.exports = function(token, options) {
 
 				return mixpanel.track(event, props, callback)
 			})
+		},
+		getDistinctId: function() {
+			if (isBrowser) {
+				return mixpanel.get_distinct_id()
+			}
+			else {
```

Comment:
NIT: let's be consistent and use
```suggestion
			} else {
```
or even drop the `else`